app/models/shelter.rb

```diff
@@ -1,2 +1,8 @@
 class Shelter < ApplicationRecord
+    validates_presence_of :name, :address, :city, :state, :zip
+    has_many :pets
+
+    def adoptable_pets
+        pets.where(adoption_status: "Adoptable")
+    end
```

Comment:
This method helps to only display the adoptable pets in the shelter pets index page